implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/v2/DynatraceExporterV2.java

```diff
@@ -360,8 +361,8 @@ private static boolean lineLengthBelowLimit(String line) {
             super(list, partitionSize);
         }
 
-        static List<List<String>> partition(List<String> list, int partitionSize) {
-            return new MetricLinePartition(list, partitionSize);
+        static List<List<String>> partition(List<String> list) {
+            return new MetricLinePartition(list, MAX_BATCH_SIZE);
```

Comment:
Is the previously used `config.batchSize()` a general config option or one that we introduced? We should honor that one and use min(config.batchSize(), MAX_BATCH_SIZE) here I suppose. What's the default if not set?

Reply:
I see. Let's do `min(config.batchSize(), MAX_BATCH_SIZE)` then.